packages/vkui/src/components/CarouselBase/CarouselBase.tsx

```diff
@@ -521,16 +528,42 @@ export const CarouselBase = ({
 
   const { isDraggable, canSlideRight, canSlideLeft } = controlElementsState;
 
+  const onScroll = (e: React.UIEvent<HTMLDivElement>) => {
+    restProps.onScroll?.(e);
+    if (rootRef.current) {
+      // eslint-disable-next-line react-compiler/react-compiler
+      rootRef.current.scrollLeft = 0;
+    }
```

Comment:
```suggestion
    event.currentColor.scrollLeft = 0;
```
чтобы превентивно поправить ошибку реакт компилятора – можем из события взять ноду

ps: ещё убрад сокращение `e` → `event`

Reply:
✅